app/controllers/concerns/idv_session.rb

```diff
@@ -1,6 +1,17 @@
 module IdvSession
   extend ActiveSupport::Concern
 
+  included do
+    before_action :sp_context_needed?
+  end
+
+  def sp_context_needed?
+    return if sp_from_sp_session.present?
+    return if LoginGov::Hostdata.env != 'prod'
```

Comment:
should this use the config from https://github.com/18F/identity-idp/pull/4545 ?

Reply:
yes, this is out of date and superceded by #4545